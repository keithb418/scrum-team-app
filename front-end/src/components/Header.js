import React from "react";
import { Row, Col } from "react-bootstrap";

const Header = () => {
  return (
    <header>
      <Row>
        <Col xs={12}>
          <div className="page-header">
            <h1>Scrum Team</h1>
          </div>
        </Col>
      </Row>
    </header>
  );
};

export default Header;
