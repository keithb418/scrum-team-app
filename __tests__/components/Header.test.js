import React from 'react';
import { shallow } from 'enzyme';
import Header from '../../front-end/src/components/Header';

test('should render the Header component correctly', () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
});