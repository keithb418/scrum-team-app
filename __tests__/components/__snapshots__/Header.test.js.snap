// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the Header component correctly 1`] = `
<header>
  <Row
    bsClass="row"
    componentClass="div"
  >
    <Col
      bsClass="col"
      componentClass="div"
      xs={12}
    />
  </Row>
</header>
`;
